docs/examples/accounts

```diff
@@ -43,6 +43,7 @@
 #    ;stunserver=stun:[user:pass]@host[:port]  # see RFC 3986: Use of the format "user:password" in the userinfo field is deprecated.
 #    ;tcpsrcport=6060
 #    ;inreq_allowed={yes, no}  # default: yes
+#    ;server_only={yes, no}  # default: no
```

Comment:
TCP and TLS already have a connection to the server, which is difficult to spoof.

What happens if you are using SIP outbound with 2 edge proxies and load balancing,
will you accept incoming request from both ?

Reply:
You're right, now the peer address is stored in the `reg` struct in BareSIP and now all of the register clients are checked (i.e. all proxies defined in outbound will be allowed). Done.